package-system/OpenSSL/set_openssl_port_to_1_1_1_m.patch

```diff
@@ -1,70 +1,63 @@
-From 66c310b442ac237b4b7e8b733eea83e94621809f Mon Sep 17 00:00:00 2001
-Date: Thu, 15 Apr 2021 09:02:46 -0700
-Subject: [PATCH] Set OpenSSL version to 1.1.1b
+From 18884b079e3945b29e4adc3b33a76c19929f9b0a Mon Sep 17 00:00:00 2001
+Date: Tue, 8 Mar 2022 12:31:34 -0800
+Subject: [PATCH] Set OpenSSL version to 1.1.1m
```

Comment:
Why do we need to patch OpenSSL?

Reply:
This looks like a patch to vcpkg (which is used to download and build), not openssl itself

Reply:
Yeah, this is a patch for vcpkg to build OpenSSL 1.1.1m.